### PR TITLE
Fixes cells icon updating

### PR DESCRIPTION
**Pull request by DominikPanic**
Fixes: #2339

### CODE REVIEW

File: code/modules/power/cell.dm [hunk 1]
```diff
@@ -6,17 +6,16 @@
...
-	overlays.Cut()
+	update_icon()
 
+/obj/item/cell/update_icon()
+	cut_overlays()
 	if(charge < 0.01)
 		return
-	else if(charge/maxcharge >=0.995)
-		overlays += image('icons/obj/power.dmi', "cell-o2")
+	else if(charge / maxcharge >= 0.995)
```
Comment (spookydonut) [inline code/modules/power/cell.dm]: Breaks blame

Comment (Rohesie) [reply code/modules/power/cell.dm]: And?

Comment (spookydonut) [reply code/modules/power/cell.dm]: just observing.

Comment (DominikPanic) [reply code/modules/power/cell.dm]: From the contributor guidelines:
```
Operators that should be separated by spaces

    Boolean and logic operators like &&, || <, >, ==, etc (but not !)
```

**Pull request merged**